common/functions/src/scalars/expressions/cast_function_template.rs

```diff
@@ -0,0 +1,20 @@
+use common_datavalues::DataType;
+use common_exception::ErrorCode;
+use common_exception::Result;
+
+use crate::scalars::CastFunction;
+use crate::scalars::Function;
+
+pub struct CastFunctionTemplate;
+
+impl CastFunctionTemplate {
```

Comment:
Why not put these into registers.

Reply:
Don't want to change original codes that not familiar with. :)